pkg/nvidiagpu/clusterpolicy.go

```diff
@@ -4,10 +4,10 @@ import (
 	"context"
 	"fmt"
 
-	nvidiagpuv1 "github.com/NVIDIA/gpu-operator/api/v1"
 	"github.com/golang/glog"
 	"github.com/openshift-kni/eco-goinfra/pkg/clients"
 	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	nvidiagpuv1 "github.com/openshift-kni/eco-goinfra/pkg/schemes/nvidiagpu/nvidiagputypes"
```

Comment:
Do you want to also move AddToScheme here?

Reply:
I'll add it to the next PR